operator/cmd/mesh/testdata/manifest-generate/input/ingressgateway_k8s_settings.yaml

```diff
@@ -13,3 +13,11 @@ spec:
           externalTrafficPolicy: Local
         serviceAnnotations:
           manifest-generate: "testserviceAnnotation"
+        securityContext:
```

Comment:
I assume the securityContext block during translation is done with replacement of what originally in the templates instead of merging, is that correct?

Reply:
Currently we're using [kubernetes strategic merge patch](https://kubernetes.io/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch/) to apply the overlay in the k8s settings. In the strategic merge patch, JSON objects are always merged but lists are replaced/merged/deleted based on **patchStrategy**.